commands/user_e2e_test.go

```diff
@@ -136,3 +140,182 @@ func (s *MmctlE2ETestSuite) TestSearchUserCmd() {
 		s.Equal("Unable to find user '"+emailArg+"'", printer.GetErrorLines()[0])
 	})
 }
+
+func (s *MmctlE2ETestSuite) TestDeleteUsersCmd() {
+	s.SetupTestHelper().InitBasic()
+
+	s.RunForSystemAdminAndLocal("Delete user", func(c client.Client) {
+		printer.Clean()
+
+		previousVal := s.th.App.Config().ServiceSettings.EnableAPIUserDeletion
+		s.th.App.UpdateConfig(func(cfg *model.Config) { *cfg.ServiceSettings.EnableAPIUserDeletion = true })
+		defer s.th.App.UpdateConfig(func(cfg *model.Config) { *cfg.ServiceSettings.EnableAPIUserDeletion = *previousVal })
+
+		cmd := &cobra.Command{}
+		confirm := true
+		cmd.Flags().BoolVar(&confirm, "confirm", confirm, "confirm")
+
+		userData := model.User{
+			Email:    s.th.GenerateTestEmail(),
+			Username: "fakeuser" + model.NewRandomString(10),
+			Password: "Pa$$word11",
+		}
+		newUser, appErr := s.th.App.CreateUser(&userData)
```

Comment:
We could just use `s.th.CreateUser` instead of `s.th.App.CreateUser`. That way we wont have to define `userData` above.

Reply:
sure, i'll adjust right away